src/main/javascript/view/ParameterView.js

```diff
@@ -7,7 +7,7 @@ SwaggerUi.Views.ParameterView = Backbone.View.extend({
 
   initialize: function(){
     Handlebars.registerHelper('isArray', function(param, opts) {
-      if (param.type.toLowerCase() === 'array' || param.allowMultiple) {
+      if (param.type && param.type.toLowerCase() === 'array' || param.allowMultiple) {
```

Comment:
This makes it safe, but you do realize, it should never get to this point, because if it does, the spec is invalid. We could add guards all over the place in the UI, but the correct place for "cleansing" invalid specs is really inside swagger-js.